### PR TITLE
feat: send transaction fees to author

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

**Comment by gregdhill** [comment]
> Can you please add a test that this is indeed doing what it is supposed to do?

@nud3l I've removed standalone fees in preparation for #363, we do not have runtime tests for the parachain runtime but I will add this when we split the "Interlay" runtime for Polkadot.

File: parachain/runtime/src/lib.rs [hunk 1]
```diff
@@ -315,8 +320,33 @@ parameter_types! {
...
+    T: pallet_authorship::Config + orml_tokens::Config,
+    GetCurrencyId: Get<T::CurrencyId>,
+{
+    fn on_unbalanceds<B>(mut fees_then_tips: impl Iterator<Item = NegativeImbalance<T, GetCurrencyId>>) {
+        if let Some(mut fees) = fees_then_tips.next() {
+            if let Some(tips) = fees_then_tips.next() {
+                tips.merge_into(&mut fees);
+            }
+            let author = pallet_authorship::Pallet::<T>::author();
+            orml_tokens::CurrencyAdapter::<T, GetCurrencyId>::resolve_creating(&author, fees);
```
[the remark this answers is not in the document]
Comment (gregdhill) [reply parachain/runtime/src/lib.rs:340]: https://docs.rs/frame-support/3.0.0/frame_support/traits/trait.Currency.html#method.resolve_creating

**Comment by nud3l** [comment]
> > Can you please add a test that this is indeed doing what it is supposed to do?
> 
> @nud3l I've removed standalone fees in preparation for #363, we do not have runtime tests for the parachain runtime but I will add this when we split the "Interlay" runtime for Polkadot.

Do you want to merge #363 before this? Just wondering if that makes sense considering the tight deadline or if we rather should tackle the manual block production after we launched KINT.

**Comment by gregdhill** [comment]
@nud3l probably not for now, but we anyway don't care about standalone fees. I would like to merge this PR ASAP though to handle collator fees.

**Review by nud3l — APPROVED** [review]

**Pull request merged**